diff --git a/node_modules/@capacitor-community/apple-sign-in/Package.swift b/node_modules/@capacitor-community/apple-sign-in/Package.swift
index 94246df..f900a1b 100644
--- a/node_modules/@capacitor-community/apple-sign-in/Package.swift
+++ b/node_modules/@capacitor-community/apple-sign-in/Package.swift
@@ -10,7 +10,7 @@ let package = Package(
             targets: ["SignInWithApple"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "7.0.0")
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", "7.0.0"..<"9.0.0")
     ],
     targets: [
         .target(
diff --git a/node_modules/@capacitor-community/apple-sign-in/android/build.gradle b/node_modules/@capacitor-community/apple-sign-in/android/build.gradle
index b045a5b..c7f021c 100644
--- a/node_modules/@capacitor-community/apple-sign-in/android/build.gradle
+++ b/node_modules/@capacitor-community/apple-sign-in/android/build.gradle
@@ -29,10 +29,10 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
-    lintOptions {
+    lint {
         abortOnError false
     }
 }
